docker/package/tezos_voting_wizard.py

```diff
@@ -129,7 +129,7 @@ def get_node_rpc_addr_query(default=None):
         validator=Validator(
             [
                 required_field_validator,
-                reachable_url_validator("chains/main/blocks/head/"),
+                reachable_url_validator("chains/main/blocks/head/header"),
```

Comment:
Are there nodes that restrict `chains/main/blocks/head`, but don't restrict `chains/main/blocks/head/header`?